lib/plugins/aws/deploy/lib/checkForChanges.js

```diff
@@ -41,7 +41,17 @@ module.exports = {
     return this.provider.request('S3',
       'listObjectsV2',
       params
-    ).then((result) => {
+    ).catch((reason) => {
+      if (!_.includes(reason.message, 'The specified bucket does not exist')) {
+        return BbPromise.reject(reason);
+      }
+      const stackName = this.provider.naming.getStackName();
+      return BbPromise.reject(new this.serverless.classes.Error([
+        `The serverless deployment bucket "${params.Bucket}" does not exist.`,
+        `Create it manually if you want to reuse the CloudFormation stack "${stackName}",`,
+        'or delete the stack if it is no longer required.',
```

Comment:
@exoego what you mean "delete the stack if it is no longer required"? 🤔

Reply:
If the deploy bucket is removed accidentally, `sls remove` can not remove the stack.
So, if user want to delete the stack which is no longer required, the stack should be manually deleted.